src/server.h

```diff
@@ -1154,6 +1155,7 @@ typedef struct ClientReplicationData {
                                            see the definition of replBufBlock. */
     size_t ref_block_pos;                /* Access position of referenced buffer block,
                                            i.e. the next offset to send. */
+    char nodeid[CLUSTER_NAMELEN];        /* Node id in cluster mode. */
```

Comment:
This is 40 bytes, unused in non-cluster mode. Should we use an allocated string instead?

If we do this, then we don't need the CLUSTER_NAMELEN definition from cluster.h.